django/contrib/gis/db/models/fields.py

```diff
@@ -371,3 +392,30 @@ class ExtentField(GeoSelectFormatMixin, Field):
 
     def get_internal_type(self):
         return "ExtentField"
+
+
+class RasterField(BaseSpatialField):
+    """
+    Raster field for GeoDjango
+    """
+
+    description = "Raster Field"
+    geom_type = 'RASTER'
+
+    def from_db_value(self, value, expression, connection, context):
+        return connection.ops.parse_raster(value)
+
+    def get_db_prep_value(self, value, connection, prepared=False):
```

Comment:
Moved the ImproperlyConfigured error into the `get_db_prep_value` function, and added tests for the exceptin in the [migrations tests](https://github.com/geodesign/django/blob/c9a5ce90aba708ea65b1263bb95fccb3e88cda75/tests/gis_tests/gis_migrations/test_operations.py#L126) and the [field tests](https://github.com/geodesign/django/blob/c9a5ce90aba708ea65b1263bb95fccb3e88cda75/tests/gis_tests/rasterapp/test_rasterfield.py#L85).